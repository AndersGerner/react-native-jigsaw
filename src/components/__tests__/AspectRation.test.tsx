import * as React from "react";
import renderer from "react-test-renderer";
import AspectRatio from "../AspectRatio.web";
import { View } from "react-native";

it("renders aspect ratio", () => {
  const tree = renderer
    .create(
      <AspectRatio style={{ aspectRatio: 2 }}>
        <View />
      </AspectRatio>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
