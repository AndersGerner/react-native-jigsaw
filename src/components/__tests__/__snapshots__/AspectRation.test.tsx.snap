// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders aspect ratio 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {},
      Object {
        "aspectRatio": 2,
      },
    ]
  }
>
  <View />
</View>
`;
